src/boards/include/boards/pimoroni_badger2040.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_PIMORONI_BADGER2040_H
+#define _BOARDS_PIMORONI_BADGER2040_H
+
+// For board detection
+#define PIMORONI_BADGER2040
+
+// --- BOARD SPECIFIC ---
+
+#ifndef BADGER2040_UART
+#define BADGER2040_UART 0
+#endif
+
+#ifndef BADGER2040_TX_PIN
+#define BADGER2040_TX_PIN 0
+#endif
+
+#ifndef BADGER2040_RX_PIN
+#define BADGER2040_RX_PIN 1
+#endif
+
+#ifndef BADGER2040_I2C
+#define BADGER2040_I2C 0
+#endif
+
+#ifndef BADGER2040_INT_PIN
+#define BADGER2040_INT_PIN 3
+#endif
+
+#ifndef BADGER2040_SDA_PIN
+#define BADGER2040_SDA_PIN 4
+#endif
+
+#ifndef BADGER2040_SCL_PIN
+#define BADGER2040_SCL_PIN 5
+#endif
+
+#ifndef BADGER2040_3V3_EN_PIN
+#define BADGER2040_3V3_EN_PIN 10
+#endif
+
+#ifndef BADGER2040_SW_DOWN_PIN
+#define BADGER2040_SW_DOWN_PIN 11
+#endif
+
+#ifndef BADGER2040_SW_A_PIN
+#define BADGER2040_SW_A_PIN 12
+#endif
+
+#ifndef BADGER2040_SW_B_PIN
+#define BADGER2040_SW_B_PIN 13
+#endif
+
+#ifndef BADGER2040_SW_C_PIN
+#define BADGER2040_SW_C_PIN 14
+#endif
+
+#ifndef BADGER2040_SW_UP_PIN
+#define BADGER2040_SW_UP_PIN 15
+#endif
+
+#ifndef BADGER2040_INKY_SPI
+#define BADGER2040_INKY_SPI 0
+#endif
+
+#ifndef BADGER2040_INKY_MISO_PIN
+#define BADGER2040_INKY_MISO_PIN 16
+#endif
+
+#ifndef BADGER2040_INKY_CSN_PIN
+#define BADGER2040_INKY_CSN_PIN 17
+#endif
+
+#ifndef BADGER2040_INKY_SCK_PIN
+#define BADGER2040_INKY_SCK_PIN 18
+#endif
+
+#ifndef BADGER2040_INKY_MOSI_PIN
+#define BADGER2040_INKY_MOSI_PIN 19
+#endif
+
+#ifndef BADGER2040_INKY_DC_PIN
+#define BADGER2040_INKY_DC_PIN 20
+#endif
+
+#ifndef BADGER2040_INKY_RESET_PIN
+#define BADGER2040_INKY_RESET_PIN 21
+#endif
+
+#ifndef BADGER2040_USER_SW_PIN
+#define BADGER2040_USER_SW_PIN 23
+#endif
+
+#ifndef BADGER2040_VBUS_DETECT_PIN
+#define BADGER2040_VBUS_DETECT_PIN 24
+#endif
+
+#ifndef BADGER2040_USER_LED_PIN
+#define BADGER2040_USER_LED_PIN 25
+#endif
+
+#ifndef BADGER2040_INKY_BUSY_PIN
+#define BADGER2040_INKY_BUSY_PIN 26
+#endif
+
+#ifndef BADGER2040_VREF_POWER_PIN
+#define BADGER2040_VREF_POWER_PIN 27
+#endif
+
+#ifndef BADGER2040_1V2_REF_PIN
+#define BADGER2040_1V2_REF_PIN 28
+#endif
+
+#ifndef BADGER2040_BAT_SENSE_PIN
+#define BADGER2040_BAT_SENSE_PIN 29
+#endif
+
+// --- UART ---
+#ifndef PICO_DEFAULT_UART
+#define PICO_DEFAULT_UART BADGER2040_UART
+#endif
+
+#ifndef PICO_DEFAULT_UART_TX_PIN
+#define PICO_DEFAULT_UART_TX_PIN BADGER2040_TX_PIN
+#endif
+
+#ifndef PICO_DEFAULT_UART_RX_PIN
+#define PICO_DEFAULT_UART_RX_PIN BADGER2040_RX_PIN
+#endif
+
+// --- LED ---
+// Included so basic examples will work, and set it to the green LED
```

Comment:
https://shop.pimoroni.com/products/badger-2040 says the LED is white? :wink:

Reply:
Removed that comment now